runtime/src/snapshot_package.rs

```diff
@@ -185,7 +166,7 @@ impl AccountsPackage {
         let accounts_db = AccountsDb::default_for_tests();
         let accounts = Accounts::new(Arc::new(accounts_db));
         Self {
-            package_kind: AccountsPackageKind::AccountsHashVerifier,
+            package_kind: AccountsPackageKind::EpochAccountsHash,
```

Comment:
Comment above says "many of the fields are invalid" so I assume it's fine to pick a random variant here?